Documentation/gettingstarted/elasticsearch.rst

```diff
@@ -29,153 +34,154 @@ The file ``es-sw-app.yaml`` will deploy the Elasticsearch service which stores S
+Fortunately, the Empire DevOps team is using Cilium for their Kubernetes cluster. Cilium provides L7 visibility and security policies to control Elasticsearch API access. Cilium follows the **white-list, least privilege model** for security. That is to say, a *CiliumNetworkPolicy* contains a list of rules that define **allowed requests** and any request that does math the rules is denied. 
 
-Since this is an ingress rule, ``fromEndpoints`` uses labels for endpoints which are calling into the *elasticsearch* service. One set of ``fromEndpoints`` are identified by labels ``name: sidious, role: lord`` (i.e. all Sith Lords named Sidious). These endpoints have both ``GET`` and ``PUT`` access as seen in the http rules section. The other ``fromEndpoints`` identified by labels ``name: vader, role: apprentice`` (i.e. all apprentices named Vader) only have ``GET`` access to specific paths including *search*.
+In this example, the policy rules are defined for inbound traffic (i.e., "ingress") connections to the *elasticsearch* service. Note that endpoints selected as backend pods for the service are defined by the *selector* labels. *Selector* labels use the same concept as Kubernetes to define a service. In this example, label ``component: elasticsearch`` defines the pods that are part of the *elasticsearch* service in Kubernetes.
+
+In the policy file below, you will see the following rules for controlling the indices access and actions performed:
+
+* ``fromEndpoints`` with labels ``app:spaceship`` only ``HTTP`` ``PUT`` is allowed on paths matching regex ``^/spaceship_diagnostics/stats/.*$``
+* ``fromEndpoints`` with labels ``app:outpost`` only ``HTTP`` ``PUT`` is allowed on paths matching regex ``^/troop_logs/log/.*$``
+* ``fromEndpoints`` with labels ``app:empire`` only ``HTTP`` ``GET`` is allowed on paths matching regex ``^/spaceship_diagnostics/_search/??.*$`` and ``^/troop_logs/search/??.*$``
+
+.. literalinclude:: ../../examples/kubernetes-es/es-sw-policy.yaml
 
 Apply this Elasticsearch-aware network security policy using ``kubectl``:
 
 .. parsed-literal::
 
     $ kubectl create -f \ |SCM_WEB|\/examples/kubernetes-es/es-sw-policy.yaml
-    ciliumnetworkpolicy "secure-empire-es" created
-
+    ciliumnetworkpolicy "secure-empire-elasticsearch" created
 
-Testing the security policy, Sidious still has both the ``GET`` and ``PUT`` access. But Vader now only has ``GET`` access and any attempts to ``PUT`` results in access denied.
+Testing the security policies, firstly, the search access is blocked for both outpost and spaceship. So from a compromised outpost, rebels will not be able to search and obtain knowledge about troops and spaceship diagnostics. Secondly, the outpost clients don't have access to create or update the ``index: spaceship_diagnostics``. 
```

Comment:
First sentence has awkward phrasing. Maybe:

"Let's test the security policies. Firstly, the search access..."